docs/data/base/components/modal/NestedModal.tsx

```diff
@@ -65,11 +65,11 @@ function ChildModal() {
     <React.Fragment>
       <Button onClick={handleOpen}>Open Child Modal</Button>
       <Modal
-        hideBackdrop
```

Comment:
The UX is broken if we disable the backdrop. I suspect this prop is here because we wanted to test that it was behaving correctly.